dev-java/openjdk/openjdk-17.0.3_p7.ebuild

```
@@ -37,11 +38,13 @@ SRC_URI="
 		$(bootstrap_uri arm64 ${ARM64_XPAK} elibc_musl)
 		$(bootstrap_uri ppc64 ${PPC64_XPAK} big-endian)
 		$(bootstrap_uri x86 ${X86_XPAK})
+		riscv? ( https://github.com/Rabenda/gentoo-riscv-bootstarp-package/releases/download/bootstrap-openjdk-17/openjdk-bootstrap-17.0.3_p7-lp64d.tar.xz )
 	)
+	riscv? ( https://github.com/Rabenda/gentoo-riscv-bootstarp-package/releases/download/bootstrap-openjdk-17/openjdk-17.0.3-riscv.patch.xz )
```

Let us use the host on my dev space:
```suggestion
	riscv? ( https://dev.gentoo.org/~arthurzam/distfiles/dev-java/openjdk/openjdk-17.0.3-riscv.patch.xz )
```